### PR TITLE
Update schedule

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

**Review by electrum — APPROVED** [review]
Nit: capitalize commit message. See https://chris.beams.io/posts/git-commit/

File: broadcast/index.md [hunk 1]
```diff
@@ -38,8 +38,22 @@ weeks!
 # Upcoming shows
  <ul>
```
Comment (electrum) [inline broadcast/index.md:39]: Can you do this list using normal Markdown instead of HTML?

Comment (bitsondatadev) [reply broadcast/index.md]: Done.

File: broadcast/index.md [hunk 2]
```diff
@@ -36,11 +36,15 @@ weeks!
...
-        Datetime: Thursday, November 19, 2020, at 1:00 pm -0500 <br/>
-        Topic(s): Hive Partitions
-    </li>
- </ul>
+ * Datetime: Thursday, December 3, 2020, at 1:00 pm -0500 <br/>
+   Topic(s): Query Planning, Remove duplicate predicates, and Memory settings
+ * Datetime: Thursday, December 17, 2020, at 1:00 pm -0500 <br/>
+   Topic(s): Cost-based optimizer  <br/>
+   Guest(s): Presto co-creator [Martin Traverso](https://twitter.com/mtraverso)
+ * Datetime: Wednesday, December 30, 2020, at 1:00 pm -0500 <br/>
```
Comment (mosabua) [inline broadcast/index.md:44]: I kind of think this is going to be a dead event .. we might want to consider skipping ..

Comment (bitsondatadev) [reply broadcast/index.md:44]: Yeah, we can update the schedule.

**Pull request merged**